### ISSUE TITLE
Announce navigation actions for screen readers

### ISSUE THREAD

**Issue by sindhuraN** [fullcalendar/fullcalendar#7274]
### Reduced Test Case

https://codepen.io/dcangulo/pen/yrQGVj

### Do you understand that if a reduced test case is not provided, we will intentionally delay triaging of your ticket?

- [X] I understand

### Which connector are you using (React/Angular/etc)?

React

### Bug Description

1. open the full calendar page (be it a demo version or anything else)
2. Switch on the Narrator on NVDA screen reader
3. navigate to the next month/prev month button using the tab key
4. on click of next month button , the calendar view moves to the next month but the screen readers are not able to recognize it , which causes an issue for visually impaired people
5.Expected Results
It should announce the month and year to which calendar view had been changed

**Comment by sindhuraN** [comment]
@acerix - Any update on this item . if you could guide us it would be helpful
